config/rbac/role.yaml

```diff
@@ -5,21 +5,30 @@ metadata:
   name: manager-role
 rules:
 - apiGroups:
-  - addon.open-cluster-management.io
+  - ""
   resources:
-  - clustermanagementaddons
-  - managedclusteraddons
+  - configmaps
+  - events
+  - pods
+  - secrets
   verbs:
   - '*'
```

Comment:
Limit the actions to just the required ones ? I don't think we require (get, delete, watch, patch, updates) for pods or events ?

Reply:
I did not add these changes. The controller-gen update did some changes to this.
I have a task created to cleanup RBACs. I'll do it soon when the features are complete.

Reply:
Alright